datasets/standvastgoed/dataset.json

```diff
@@ -612,6 +601,75 @@
         },
         "mainGeometry": "adresseerbaarobjectGeometrie"
       }
+    },
+    {
+      "id": "eigendom",
+      "type": "table",
+      "version": "1.0.0",
+      "auth": "FP/MDW",
+      "reasonsNonPublic":"5.1 1c: Bevat persoonsgegevens",
+      "title": "Eigendomsverhouding, type eigenaar en grootte bezit van eigenaar per verblijfsobject per 1 januari peildatum",
+      "description": "Alleen verblijfsobjecten met status 'Verblijfsobject in gebruik','Verblijfsobject in gebruik (niet ingemeten)' of 'Verbouwing verblijfsobject'. Afgeleide gegevens uit een combinatie van bronnen: BRK, BAG, BRP, BWV, WOZ, HR en AFWC. Inclusief historie, met data vanaf 2021. Publicatie van nieuwe 1 januari stand met eigendomsverhouding gebeurt omstreeks juli.",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "identifier": [
+          "id"
+        ],
+        "required": [
+          "schema",
+          "id"
+        ],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "string",
+            "title": "id",
+            "description": "Unieke identificatie voor dit object op basis van adresseerbaar object identificatie en peildatum"
+          },
+          "peildatum": {
+            "type": "string",
+            "format": "date",
+            "title": "peildatum",
+            "description": "Peildatum van eigendomsverhouding"
+          },
+          "adresseerbaarobjectIdentificatie": {
+            "type": "string",
+            "title": "adresseerbaarobjectIdentificatie",
+            "shortname": "aotID",
```

Comment:
Dit zal op postgres db niveau dus `aot_i_d` worden. Het is beter er `aotId` van te maken, dan wordt het in postgres `aot_id`.

Reply:
fixed